src/components/inputs/multi-select/multi-select.component.tsx

```diff
@@ -131,7 +131,7 @@ const MultiSelect: React.FC<FormFieldInputProps> = ({ field, value, errors, warn
           </Layer>
         </div>
         <div className={styles.selectionDisplay}>
-          {value?.length && field.questionOptions.answers?.length > 5 ? (
+          {value?.length ? (
```

Comment:
Thanks @samuelmale. Question though, doesn't this introduce the selected tags also for the multi-select checkboxes? IMO, we may not need to see the tags for the checkboxes since it is already clear which items are selected at first glance.

Reply:
Updated:

<img width="255" alt="Screenshot 2024-08-13 at 10 21 10" src="https://github.com/user-attachments/assets/df3e6acc-1ce4-4f0e-bf7c-d6a71dff58ce">